### PR TITLE
New cli command modify-user & saltcorn-install improvement & more jsdoc & small improvements

### PR DESCRIPTION
**Pull request by AleksandrSokolov**

### CODE REVIEW

**Comment by AleksandrSokolov** [comment]
Also make some improvements in install  script (npx saltcorn-install). 
Adds:

- -e, --expert Expert mode, more abilities for configuration (Not compatible with -y)
- -d, --dryrun Dry Run mode, displays the operations that would be performed using the specified command without actually running them

File: packages/saltcorn-cli/src/commands/create-user.js [hunk 1]
```diff
@@ -5,7 +5,11 @@
 const { Command, flags } = require("@oclif/command");
 const { cli } = require("cli-ux");
 const { maybe_as_tenant } = require("../common");
+const { loadAllPlugins } = require("@saltcorn/server/load_plugins");
```
Comment (glutamate) [inline packages/saltcorn-cli/src/commands/create-user.js:8]: Imports that import the database module (or does so transitively e.g. server) and therefore initialise the database connection should be done inside the methods. I can't remember exactly what it is that goes wrong if you don't do this but one of the commands requires this. having the imports outside the class declaration in just one cli command means that the database will be connected in all of the commands before running. 

This is also why the functions in common do their imports inside the functions.

However you can make this much shorter by using init_some_tenants. this was only recently introduced and has not been implemented in all the commands but have a look at configuration-check as an example

File: packages/saltcorn-cli/src/commands/modify-user.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
+/**
+ * @category saltcorn-cli
+ * @module commands/modify-user
+ */
+
+// todo support for  users without emails (using user.id)
+const { Command, flags } = require("@oclif/command");
+const { cli } = require("cli-ux");
+const { maybe_as_tenant } = require("../common");
+const {loadAllPlugins} = require("@saltcorn/server/load_plugins");
```
Comment (glutamate) [inline packages/saltcorn-cli/src/commands/modify-user.js:10]: see comment on create-user; move imports inside methods

**Review by glutamate — CHANGES_REQUESTED** [review]
thank you for fixing many bugs here. see the comments, only small changes

File: packages/saltcorn-cli/src/commands/modify-user.js (hunk 2 above)
Comment (AleksandrSokolov) [reply packages/saltcorn-cli/src/commands/modify-user.js:10]: Updated  according review

**Comment by AleksandrSokolov** [comment]
Update according @glutamate review. Fix package.json

File: packages/saltcorn-cli/src/commands/modify-user.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
+/**
+ * @category saltcorn-cli
+ * @module commands/modify-user
+ */
+
+// todo support for  users without emails (using user.id)
+const { Command, flags } = require("@oclif/command");
+const { cli } = require("cli-ux");
+const { maybe_as_tenant, init_some_tenants } = require("../common");
+const User = require("@saltcorn/data/models/user");
```
Comment (glutamate) [inline packages/saltcorn-cli/src/commands/modify-user.js:10]: i think you forgot to delete this line (needs to be inside, which it is)

**Review by glutamate — CHANGES_REQUESTED** [review]
one change - delete import

**Review by glutamate — APPROVED** [review]

**Pull request merged**